health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/AppFieldScreenWrapper.js

```
@@ -0,0 +1,215 @@
+                };
+              }
+              return {
+                ...j,
+                active: false,
+              };
+            });
+          });
+        }}
+      />
+      <Stepper
+        customSteps={[...stepper?.map((i) => i.name)]}
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)